docs/usage/RsyncConfiguration.md

```diff
@@ -90,6 +90,72 @@ The above will set the retry limit to `40`.
 
 ---
 
+### Configuring Rsync command using Migration CR
+
+MTC 1.7.5 onwards, user can choose to run rsync pods as root, or a specific user/group, or with the default uid/gid allowed from the uid/gid range of a namespace. By default the rsync pod will run with the minimal capabilities and with the default uid/gid provided by SCC controller at run time.
+
+MTC 1.7.5 introduces three fields in migmigration CR to allow users to use different pod profile:
+
+- `RunAsRoot` - By default the value is not set and field is omitted. If passed `true`, rsync pod will run with `privilege` SCC. Takes precedence over below mentioned two fields.
```

Comment:
`privileged` not `privilege`

$ oc get scc 
NAME         PRIV   CAPS    SELINUX    RUNASUSER   FSGROUP    SUPGROUP   PRIORITY     READONLYROOTFS   VOLUMES
privileged   true   ["*"]   RunAsAny   RunAsAny    RunAsAny   RunAsAny   <no value>   false            ["*"]

Other than that the changes all look good to me.